hibernate-core/src/main/java/org/hibernate/internal/EntityManagerMessageLogger.java

```diff
@@ -133,4 +133,12 @@ void unableToLocateStaticMetamodelField(
 	)
 	void duplicatedPersistenceUnitName(String name);
 
+	@LogMessage(level = WARN)
```

Comment:
Please use `DeprecationLogger` for this..

Reply:
Fixed, thanks!